### PR TITLE
TFRS - new input field for organization details - IDIR only#2579

### PR DESCRIPTION
**Pull request by jig-patel**
TFRS - new input field for organization details - IDIR only#2579

### CODE REVIEW

File: backend/api/migrations/0009_organization_edrms_record.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.20 on 2023-10-10 22:21
+
+from django.db import migrations, models
+
+def change_organization_types(apps, schema_editor):
```
Comment (AlexZorkin) [inline backend/api/migrations/0009_organization_edrms_record.py:5]: Is this supposed to be here? I don't see this as part of ticket #2579

Comment (jig-patel) [reply backend/api/migrations/0009_organization_edrms_record.py:5]: if you look at the wireframe, supplier Type radio button label changed.
https://preview.uxpin.com/59145260c7d8cc48969ac79401cc03d84843c3d9#/pages/161516664/simulate/no-panels?mode=mf

File: backend/api/viewsets/Organization.py [hunk 2]
```diff
@@ -80,7 +80,7 @@ def list(self, request, *args, **kwargs):
         data = cached_page.get(sanitized_cache_key)
         if data is not None:
             return Response(data)
-        fuel_suppliers = Organization.objects.select_related('type').filter(type__type='Part3FuelSupplier').order_by('id')
+        fuel_suppliers = Organization.objects.select_related('type').filter(type__type='FuelSupplier').order_by('id')
```
Comment (AlexZorkin) [inline backend/api/viewsets/Organization.py:83]: I think this has to be reverted as well

Comment (jig-patel) [reply backend/api/viewsets/Organization.py]: done

**Review by AlexZorkin — APPROVED** [review]

**Pull request merged**